views/default/notifier/notifier.css

```diff
@@ -0,0 +1,33 @@
+#notifier-popup { padding: 0; background: white; width: 345px; position: absolute; font-size: 14px; line-height: normal; }
+#notifier-popup .elgg-pagination { display: none !important; }
+#notifier-popup .elgg-no-results, #notifier-popup .elgg-ajax-loader { margin: 0; padding: 8px; }
+
+.notifier-none { display: block; text-align: center; padding: 8px; }
+
+.elgg-list-notifier { max-height: 300px; overflow-y: auto; margin: 0; }
+.elgg-list-notifier > li { padding: 0; border: none; box-shadow: none; border-bottom: 1px solid #eee; margin: 0; }
+.elgg-list-notifier > li:last-of-type { border-bottom-width: 0; }
+.elgg-list-notifier .elgg-image-block { padding: 8px;  transition: background-color 0.5s ease; }
+.elgg-list-notifier .elgg-image-block.elgg-notifier-unread { background: rgba(238, 238, 238, 0.5); }
+.elgg-list-notifier .elgg-image-block:after { content: ""; }
+
+#notifier-popup .elgg-foot { border-top: 1px solid #eee; display: flex; align-items: center; }
+#notifier-popup .elgg-foot > div { flex: 0 0 auto; text-align: center; font-size: 12px; min-width: 30px; padding: 4px; }
+#notifier-popup .elgg-foot > div:nth-of-type(2) { flex: 1 1 auto; }
+#notifier-popup .elgg-foot a { display: inline-block; width: 100%; text-align: center; }
+#notifier-popup .elgg-foot a.hidden { display: none; }
+
+.elgg-menu-item-notifier > a { position: relative; }
+.elgg-menu-item-notifier > a .elgg-icon { font-size: 20px; }
+
+#notifier-new { color: white; background-color: red; -webkit-border-radius: 10px; -moz-border-radius: 10px; border-radius: 10px; -moz-box-shadow: none; -webkit-box-shadow: none; box-shadow: none; position: absolute; text-align: center; top: 2px; left: auto; right: -8px; min-width: 16px; height: 16px; line-height: 16px; font-size: 10px; font-weight: bold; }
+
+/* Lightbox */
+.notifier-lightbox-wrapper { width: 570px; min-height: 410px; margin: 20px 15px; }
+
+#notificationstable td.notifiertogglefield { width: 50px; text-align: center; vertical-align: middle; }
+#notificationstable td.notifiertogglefield input { margin-right: 10px; }
+#notificationstable td.notifiertogglefield a { display: inline-block; font-family: 'FontAwesome'; font-size: 18px; line-height: 20px; vertical-align: middle; text-rendering: auto; -webkit-font-smoothing: antialiased; -moz-osx-font-smoothing: grayscale; }
+#notificationstable td.notifiertogglefield a:after { content: "\f0ac"; }
+#notificationstable td.notifiertogglefield a.notifiertoggleOff { color: #cccccc; }
+#notificationstable td.notifiertogglefield a.notifiertoggleOn { color: #4078c0; }
```

Comment:
Could you revert this to the original formatting? If possible, also make separate commits for moving the file and modifying contents, so it's easier to see what has changed.